src/WallpaperEngine/Core/CProject.cpp

```diff
@@ -20,11 +22,23 @@ CProject* CProject::fromFile (const irr::io::path& filename)
     auto type = jsonFindRequired (content, "type", "Project type missing");
     auto file = jsonFindRequired (content, "file", "Project's main file missing");
     auto general = content.find ("general");
+    CWallpaper* wallpaper;
+
+    if (strcmp ((*type).get <std::string> ().c_str (), "scene") == 0)
```

Comment:
Do not use C functions for strings, *type can be directly compared with "==" to "scene", it'll take care of the casting to std::string and call the proper function for comparing them.

Reply:
This still needs changing